ee/clickhouse/sql/clickhouse.py

```diff
@@ -46,23 +40,25 @@
 REPLACING_MERGE_TREE = "replacing_merge_tree"
 
 
-def table_engine(table: str, ver: Optional[str] = None, engine_type: Optional[str] = None) -> str:
+def table_engine(table: str, ver: str, engine_type: str) -> str:
     if engine_type == COLLAPSING_MERGE_TREE and ver:
-        return COLLAPSING_TABLE_ENGINE.format(table=table, ver=ver)
+        return COLLAPSING_TABLE_ENGINE().format(table=table, ver=ver)
     elif engine_type == REPLACING_MERGE_TREE and ver:
-        return REPLACING_TABLE_ENGINE.format(table=table, ver=ver)
+        return REPLACING_TABLE_ENGINE().format(table=table, ver=ver)
     else:
-        return MERGE_TABLE_ENGINE.format(table=table)
+        raise ValueError(f"Unknown engine type {engine_type}")
 
 
 def kafka_engine(
     topic: str,
-    kafka_host=KAFKA_HOSTS,
+    kafka_host=None,
     group="group1",
     serialization="JSONEachRow",
     proto_schema=None,
     skip_broken_messages=100,
 ):
+    if kafka_host is None:
```

Comment:
legitimately curious: why?

Reply:
So we'd be able to override it in tests. Premature optimization from me, didn't need to be here.